### PR TITLE
nrfx: hal: Add missing nrf_aar_int_enable function

### PR DESCRIPTION
**Pull request by cvinayak**
Add the missing nrf_aar_int_enable function.

Signed-off-by: Vinayak Kariappa Chettimada <vich@nordicsemi.no>

### CODE REVIEW

File: src/nrfx/hal/nrf_aar.h [hunk 1]
```diff
@@ -84,6 +84,14 @@ NRF_STATIC_INLINE bool nrf_aar_event_check(NRF_AAR_Type const * p_reg,
 NRF_STATIC_INLINE void nrf_aar_event_clear(NRF_AAR_Type *  p_reg,
                                            nrf_aar_event_t event);
 
+/**
+ * @brief Function for enabling the specified interrupts.
+ *
+ * @param[in] p_reg Pointer to the structure of registers of the peripheral.
+ * @param[in] mask  Mask of interrupts to be enabled.
+ */
+void nrf_aar_int_enable(NRF_AAR_Type * p_reg, uint32_t mask);
```
Comment (rugeGerritsen) [inline src/nrfx/hal/nrf_aar.h:93]: Use `NRF_STATIC_INLINE ` to mimize the diff with the real nrf_aar.h from nrfx

**Review by rugeGerritsen — APPROVED** [review]

File: src/nrfx/hal/nrf_aar.h (hunk 1 above)
Comment (cvinayak) [reply src/nrfx/hal/nrf_aar.h:93]: @rugeGerritsen Could you please take the commits in this PR into your PR#13? This way the changes here can be made in sync with nrfx. This PR can then be closed.

Comment (rugeGerritsen) [reply src/nrfx/hal/nrf_aar.h:93]: Done

**Comment by cvinayak** [comment]
Superceded by #13

**Pull request closed without merging**